ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/DefendantMapper.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.hmcts.cmc.ccd.mapper;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.ccd.domain.CCDDefendant;
+import uk.gov.hmcts.cmc.ccd.domain.CCDPartyType;
+import uk.gov.hmcts.cmc.ccd.exception.MappingException;
+import uk.gov.hmcts.cmc.domain.models.otherparty.CompanyDetails;
+import uk.gov.hmcts.cmc.domain.models.otherparty.IndividualDetails;
+import uk.gov.hmcts.cmc.domain.models.otherparty.OrganisationDetails;
+import uk.gov.hmcts.cmc.domain.models.otherparty.SoleTraderDetails;
+import uk.gov.hmcts.cmc.domain.models.otherparty.TheirDetails;
+
+import java.time.LocalDate;
+import java.util.Optional;
+
+@Component
+public class DefendantMapper {
```

Comment:
Please rename it to `TheirDetailsMapper` or `ClaimantProvidedTheirDetailsMapper`.